### ISSUE TITLE
Pico: Definition of ack>/ack< missing after latest commit

### ISSUE THREAD

**Issue by anarchitech** [nimblemachines/muforth#24]
commit 33394be397eb49269e03876d81fe43467f2a6678 removed the definition of ack>/ack< from target/ARM/debug/picoboot.mu4

Until fixed, patch by hand, insert below definition of pico>

```
( Transferring data)
: ack> pad 0 uread ;
: ack< pad 0 uwrite;
```

**Comment by anarchitech** [comment]
If you change values to 1, you will get joy on OpenBSD and be able to chat with the target:

```
( Transferring data)
: ack> pad 1 uread ;
: ack< pad 1 uwrite;
```

**Comment by anarchitech** [comment]
Fixed by commit 2a9fea0
